The origin cargo.toml file call a local version of proj-sys crate. With this patch we
use the guix packaged version of this lib.

diff --git a/Cargo.toml b/Cargo.toml
index 24f0198..cb079b3 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,7 +30,7 @@ subprocess = "0.2.9"
 rayon = "1.6.1"
 # crates needed for the proj adaptation
 #proj-sys = { version = "0.23.1" , features = ["network"]}
-proj-sys = { version = "0.23.1", features = ["network"], path = "proj/proj-sys" }
+proj-sys = { version = "0.23.1", features = ["network"]}
 num-traits = "0.2.14"
 libc = "0.2.119"
 thiserror = "1.0.30"
